<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTabDySetDlg</class>
 <widget class="QWidget" name="CTabDySetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTabDySetDlg</string>
  </property>
  <widget class="QPushButton" name="BTN_SAVE">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10000以下</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10000以上（包含10000）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDIT_W_DOWN">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDIT_W_UP">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
